packages/core/test/overlay/overlayTests.tsx

```diff
@@ -19,6 +19,7 @@ describe("<Overlay>", () => {
     let wrapper: ReactWrapper<IOverlayProps, any>;
 
     afterEach(() => {
+        document.body.classList.remove(Classes.OVERLAY_OPEN);
```

Comment:
why in the test? i'd expect the component to handle removing the class. 
probably means you need to also be removing it in `componentWillUnmount()`